app/src/main/java/ch/epfl/qedit/view/quiz/QuizActivity.java

```diff
@@ -80,6 +86,34 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
             case android.R.id.home:
                 onBackPressed();
                 break;
+            case R.id.validate:
+                new AlertDialog.Builder(this)
```

Comment:
I think that we have already created a `ConfirmDialog` in the package `view.util`. If you reuse it, it would probably make that big piece of code much shorter ;) .

Reply:
Yes, thank you !